Rust/src/rcf.rs

```diff
@@ -662,42 +662,42 @@ where
             visitor_info,
         );
         let field_summarizer = FieldSummarizer::new(centrality, project, max_number, l1distance);
-        field_summarizer.summarize_list(&self.point_store, &raw_list, &new_positions)
+        Ok(field_summarizer.summarize_list(&self.point_store, &raw_list, &new_positions))
```

Comment:
This will probably be edited in the next few rounds -- but would the unwrap() be automatic within the OK()?